www/index.html

```diff
@@ -167,12 +167,13 @@
       <div id="footer">
         <div class="menu-group">
           <div class="menu-group-left"></div>
+          <div class="presenter-switch"> </div>
           <!-- <div class="menu-group-right">
-            <ul class="menu-bar">
+             <ul class="menu-bar">
```

Comment:
commented code should be removed eventually.

Reply:
Yeah, I agree. I will remove it.